kubejs/server_scripts/base/tags/items/emendatusenigmatica/gold.js

```diff
@@ -1,3 +1,10 @@
 ServerEvents.tags('item', (event) => {
```

Comment:
This file should be in `base/tags/items/minecraft/piglin_loved.js`

Reply:
Why did I name it gold in the first place 🤔